# quarTeT: Telomere-to-telomere Toolkit
quarTeT is a collection of tools for T2T genome assembly and basic analysis in automatic workflow.

Task include:
- [AssemblyMapper](#AssemblyMapper): reference-guided genome assembly
- [GapFiller](#GapFiller): long-reads based gap filling
- [CentroMiner](#CentroMiner): centromere candidate prediction
- [TeloExplorer](#TeloExplorer): telomere identification

## Getting Started
quarTeT can be easily accessed via our web server. Hover your mouse on Tools tab, and click corresponding module to start.

## General
All modules have a similar structure.
### Uploading Data
You can drag or click to upload files.

If you access quarTeT from [443(https) port](https://atcgn.com/quarTeT/pages/home.html), uploading may be slow. 

Access quarTeT from [8080 port](http://atcgn.com:8080/quarTeT/pages/home.html) may accel uploading.
### Setting Algorithm Parameters
We have set default paramater which may solve most data. 

If you are unsure of these parameter, just try default. You can click `reset` at the bottom right to get back default parameter.

Some paramater with bar can only switch in a window. However, you can directly input in box at the right. You will receive a warning if input is too large or small.
### Submit Task and Access Result
Click `Run` at the bottom right to submit task. 

If all checking pass, you will be asked to leave your e-mail. Once task is completed, we will send you the task query ID. 

You can access your result from top of home page via query ID.

Some task may be processed with warning messages, you can check them at the top right of result page.

Click download icon/buttom will open a new page. If download dialog doesn't pop up, right click the page and select `Save as ...` to download.

For plots, you can click the magnifier icon beside the plot to enlarge it. 

For statistic tables, you can drag to resize tabs.

## AssemblyMapper
AssemblyMapper is a reference-guided assemble tool.
### Uploading Sequence
You should upload 2 files in this section.
- Reference genome (FASTA format)

  a high-quality genome of close-related species.
- Contigs (FASTA format)

  A haplotype-resloved contig-level assembly.

  It's recommended to obtain such an assembly using hifiasm (https://github.com/chhylp123/hifiasm).

  you can convert `{prefix}.bp.hap1.p_ctg.gfa` and `{prefix}.bp.hap2.p_ctg.gfa` generated by hifiasm to fasta format as input, separately.
### Setting Algorithm Parameters
6 parameters is availble on web for this module.
- Min Contigs Length

  Contigs shorter than INT (bp) will be removed, default: 50000
- Min Alignment Length

  The min alignment length to be select (bp), default: 10000
- Min Alignment Identity

  The min alignment identity to be select (%), default: 90
- Aligner

  Specify alignment program (support minimap2 and mummer), default: minimap2
- Plot

  Plot a colinearity graph for draft genome to reference alignments. (will cost more time)
- Filename

  The name (prefix) used on your generated files. Default: quarTeT
### Result
