# quarTeT: Telomere-to-telomere Toolkit
quarTeT is a collection of tools for T2T genome assembly and basic analysis in automatic workflow.

Task include:
- [AssemblyMapper](#AssemblyMapper): reference-guided genome assembly
- [GapFiller](#GapFiller): long-reads based gap filling
- [CentroMiner](#CentroMiner): centromere candidate prediction
- [TeloExplorer](#TeloExplorer): telomere identification

## Getting Started
quarTeT can be easily accessed via our web server. Hover your mouse on Tools tab, and click corresponding module to start.

## General
All modules have a similar structure.
### Uploading
If you access quarTeT from [443(https) port](https://atcgn.com/quarTeT/pages/home.html), uploading may be slow. 

Access quarTeT from [8080 port](http://atcgn.com:8080/quarTeT/pages/home.html) may accel uploading.
### Setting Parameters

### Access Result


## AssemblyMapper
AssemblyMapper is a reference-guided assemble tool.
### Uploading Sequence
You should upload 2 files in this section.
- Reference genome (FASTA format)

  a high-quality genome of close-related species.
- Contigs (FASTA format)

  A haplotype-resloved contig-level assembly.

  It's recommended to obtain such an assembly using hifiasm (https://github.com/chhylp123/hifiasm).

  you can convert `{prefix}.bp.hap1.p_ctg.gfa` and `{prefix}.bp.hap2.p_ctg.gfa` generated by hifiasm to fasta format as input, separately.
### Setting Algorithm Parameters
We have set default paramater which may solve most data. 

If you are unsure of these parameter, just try default.

Some paramater with bar can only switch in a window. However, you can directly input in box at the right.